frontend/src/components/Contexts/MissionListsContext.tsx

```diff
@@ -114,7 +114,7 @@ export const useMissions = (): MissionsResult => {
                 })
             })
         }
-    }, [registerEvent, connectionReady])
+    }, [registerEvent, connectionReady, missionQueue])
```

Comment:
Why has there been changes in this file?

Reply:
Because it caused a warning, and I dont think this create a circular dependency, so I guess it should be there.